drivers/SmartThings/matter-sensor/profiles/rain-battery-fault.yml

```diff
@@ -0,0 +1,14 @@
+name: rain-battery-fault
```

Comment:
I noticed that the PR flagged this profile as a duplicate of rain-fault. Is this profile missing the battery capability?

Reply:
good catch Nick! Thank you